README.md

```diff
@@ -73,48 +68,26 @@ After you're finished please remove all the comments and instructions!
 > Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] While you did a good job following all the instructions on your README :+1:, it is required to remove all the section descriptions, like:
> Describe the tech stack, and include only the relevant sections that apply to your project.

Kindy checks your README and removes each of them. It would make your project looks more professional